src/__tests__/testcases/shadowing/index.ts

```diff
@@ -1 +1,8 @@
-export { GenericInterface, GenericKlass, genericFunction, ConditionalInfer, Mapped } from "./exports";
+export {
+  GenericInterface,
+  GenericKlass,
+  genericFunction,
+  ConditionalInfer,
+  Mapped,
+  GenericType
```

Comment:
This one was missing, so it wasn't actually being tested. Luckily, it works fine. 😅